packages/eslint-config-hudl/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+
+### 6.0.0 / 2018-01-18
+- [breaking] Updated indentation for switch statements
+- [minor] allow multiple stateless react components in a single file
```

Comment:
Hey. What are the use cases of this that you are seeing?

Reply:
we normally write some helper functions in react that will return a snippet of jsx to help break apart of the logic. With the current setup, eslint will complain about that. There is a way to get around it (by assigning the jsx to a variable, then returning the variable) that people do already so I figured we might as well make it more standard. By limiting this to only multiple stateless functions in a single I think that is acceptable to work with.